contribs/gmf/less/search.less

```diff
@@ -108,6 +108,9 @@ gmf-search {
       }
     }
   }
+  .tt-menu.tt-open.gmf-search-no-results {
+    display: block !important;
```

Comment:
We shouldn't use !important ...

Reply:
I know but it's a hackish workaround for a typeahead limitation that will never be fixed upstream